### ISSUE TITLE
Double classes with markdown extra

### ISSUE THREAD

**Issue by jlehtinen** [OleVik/grav-plugin-imgcaptions#11]
When I use markdown extra to add image classes:

`![](app.jpg){.hero-image}`

what I get is 

~~~html
<figure role="group" class="hero-image">
  <img alt="" class="hero-image" src="grav-image-path/app.jpeg">
</figure>
~~~

Would be handy in most cases if it didn't include the class on both elements. Is this something that could be easily fixed?

**Comment by OleVik** [comment]
Not easily, as the engine that runs the plugin is Regex-based, but I will look at a possible solution.

**Comment by OleVik** [comment]
Should be solved by Markdown-mode in ![v2.0.0-beta1](https://github.com/OleVik/grav-plugin-imgcaptions/releases/tag/v2.0.0-beta1).

**Comment by jlehtinen** [comment]
That's just great! Many thanks.
